operators/test/e2e/helpers/elasticsearch.go

```diff
@@ -19,10 +19,6 @@ import (
 // if `--auto-port-forward` is passed to `go test`, then use a custom
```

Comment:
Should this comment be removed or moved into `params`?

Reply:
The flag description is quite descriptive.
https://github.com/elastic/cloud-on-k8s/blob/7a3f58d144deeeacf564e725274b3cf8ba59d296/operators/test/e2e/params/params.go#L29-L30
Let's remove this comment.